### DISCUSSION TITLE
Displaying published posts count

### DISCUSSION THREAD

**Discussion by ArulrajAboo** [forem/forem#19486] [Q&A]
Hi There,

On the organization page, the published count always displays 8. Is that an issue? Please check the following,
https://dev.to/boldreports
There are more than 8 posts in the organization but still shows 8.
![image](https://github.com/forem/forem/assets/95745238/f34f71aa-af76-4a7f-8b7f-5e644a86c9f9)

Reg,
Arulraj A

**Comment by maestromac** [comment]
This may have been an caching issue. I see that it's been resolved
